python-project-template/.github/workflows/testing-and-coverage.yml.jinja

```diff
@@ -1,3 +1,4 @@
+{%- import 'python-versions.jinja' as py %}
```

Comment:
I think I don't understand Jinja here. You import `python-versions.jinja` as `py`, but bellow you call `py.min(python_versions)`. Is it the same `py`?

Reply:
Yes - that's the same py. It's a set of custom macros for populating appropriate python versions in various spots.

You can read more about its use in the header comment: https://github.com/lincc-frameworks/python-project-template/blob/main/python-versions.jinja